app/controllers/rounds_controller.rb

```diff
@@ -1,7 +1,10 @@
 class RoundsController < ApplicationController
   def public_show
     service = UsersService.new
-    @current_round_info = service.current_round
+    response = service.current_round
+    current_round_info = Round.new(response[:data][:attributes])
+    current_round_info = JSON.parse(current_round_info.body, symbolize_names: true)
```

Comment:
maybe I should move the json parse method out so it can be used here, or just remove it entirely from application_serializer and we do it in each location.